test/home_screen/home_screen_test.dart

```diff
@@ -18,18 +21,24 @@ import 'package:mockito/mockito.dart';
 class MockNavigatorObserver extends Mock implements NavigatorObserver {}
 
 void main() {
+  TasksRepository tasksRepository;
+
   Future<void> initHomeScreen(WidgetTester tester, [Size size = samsungGalaxyNote5]) async {
     await tester.pumpWidget(
-      ScreenTestingWrapper(
-        screenSize: size,
-        screen: HomeScreen(),
+      BlocProvider(
+        create: (context) => TasksBloc(tasksRepository: tasksRepository),
+        child: ScreenTestingWrapper(
+          screenSize: size,
+          screen: HomeScreen(),
+        ),
       ),
     );
   }
 
   group('HomeScreen', () {
     setUp(() {
       setFirestoreInstance(MockFirestoreInstance());
+      tasksRepository = TasksRepository();
```

Comment:
If these are in the wrong order, the test will fail without saying why.  Maybe add some logic to throw an error if no firestore instance can be found.

Or maybe make a test method `initFirestoreForTasks` or something

Reply:
We could put it all in one method, but we should not be doing more configuration than is necessary or else our unit tests will be bad.  So just leaving it like this is okay.  Also we get an error if the instance has not been set, so that should be enough